classes/user.rb

```diff
@@ -0,0 +1,22 @@
+# The User class describes an account holder.
+class User
+  attr_accessor :name, :pin, :balance
+
+  def initialize
+    @name = ""
+    @pin = ""
+    @balance = 0
+  end
+
+  # Verify that the user can withdraw the requested amount
+  def can_withdraw?(amount)
+    amount.class == Fixnum && @balance >= amount && amount > 0
+  end
```

Comment:
given this is in 2 places, I would consider using a `module`/mixin to share code